.prettierignore

```diff
@@ -138,6 +141,9 @@ seed-users.json
 # Storybook
 **/storybook-static
 
+# Status
+/libs/status/crate/src/gen
+
```

Comment:
```suggestion
```

Reply:
This is removed in https://github.com/hashintel/hash/pull/1810